skimage/filters/thresholding.py

```diff
@@ -881,3 +881,42 @@ def threshold_sauvola(image, window_size=15, k=0.2, r=None):
         r = 0.5 * (imax - imin)
     m, s = _mean_std(image, window_size)
     return m * (1 + k * ((s / r) - 1))
+
+
+def apply_hysteresis_threshold(image, low, high):
```

Comment:
The `apply_hysteresis_threshold` name is a little long and seems a bit inconsistent with the other thresholding routines which are all named `threshold_XXX`.  I am thinking specifically of interactive use from Ipython where a user may hit `skimage.filters.<tab>` and would not see it grouped with the other thresholding routines.  I see that most of those threshold routines return a float corresponding to a threshold rather than actually threholding an image, but `threshold_niblack` and `threshold_sauvola` do return threshold masks.